### PR TITLE
Add lamarzocco bluetooth docs

### PR DESCRIPTION
**Pull request by zweckj**
## Proposed change

Add lamarzocco bluetooth docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/108287
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lamarzocco.markdown [hunk 1]
```diff
@@ -18,15 +18,18 @@ ha_platforms:
...
   - switch
   - update
+ha_bluetooth: true
 ha_codeowners:
   - '@zweckj'
 ha_integration_type: device
 ---
 
-This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API and (optionally) local API calls, which include a WebSocket connection for (near) real-time updates. 
+This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API. Optionally local API calls, which include a WebSocket connection for (near) real-time updates and a Bluetooth connection can be utilized for local connections.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
This integration interacts with [La Marzocco coffee machines](https://lamarzocco.com/it/en/) through calls to the LaMarzocco cloud API. Optionally, local API calls, which include a WebSocket connection for (near) real-time updates and a Bluetooth connection, can be utilized for local connections.
```

File: source/_integrations/lamarzocco.markdown [hunk 2]
```diff
@@ -18,15 +18,18 @@ ha_platforms:
...
 
 To be able to configure your machine in Home Assistant, your machine needs to be added to your account using the official La Marzocco app first. Currently, only login with username & password is supported. If you are currently using a social login, you need to create a new LaMarzocco account and transfer your machine to it to be able to use this integration.
 
+If your machine is in Bluetooth range to your Home Assistant host the machine will be discovered automatically.
```
Comment (c0ffeeca7) [inline source/_integrations/lamarzocco.markdown]:
```suggestion
If your machine is in Bluetooth range to your Home Assistant host, the machine will be discovered automatically.
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @zweckj :+1:

**Comment by zweckj** [comment]
> Thank you, @zweckj 👍

sorry just added a ref to the BT integration the moment you approved 🤦‍♂️

**Comment by c0ffeeca7** [comment]
> > Thank you, @zweckj 👍
> 
> sorry just added a ref to the BT integration the moment you approved 🤦‍♂️


You want me to let this sit for a couple of hours? Still drafting?

**Comment by zweckj** [comment]
> > > Thank you, @zweckj 👍
> > 
> > 
> > sorry just added a ref to the BT integration the moment you approved 🤦‍♂️
> 
> You want me to let this sit for a couple of hours? Still drafting?

no I'm done. Was just checking if I missed anything from the other integrations.

**Review by c0ffeeca7 — APPROVED** [review]

**Pull request merged**